src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.ts

```diff
@@ -249,6 +249,6 @@ export class HealthComponent implements OnInit, OnDestroy {
       return 0;
     }
 
-    return Math.round((dividend / divisor) * 100);
+    return Math.ceil((dividend / divisor) * 100);
```

Comment:
As we're presenting 2 decimal places in the legend, let's be consistent with that:
```suggestion
    return Math.ceil((dividend / divisor) * 100 * 100) / 100;
```